backend/src/main/java/com/redhat/cloud/notifications/routers/models/RenderEmailTemplateRequest.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.cloud.notifications.routers.models;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+public class RenderEmailTemplateRequest {
```

Comment:
This probably needs `@JsonNaming(SnakeCaseStrategy.class)` and `@Schema` annotations on the fields for the OpenAPI output.

Reply:
Yes, we need the SnakeCaseStrategy. Not sure why we need the `@Schema` 
 https://github.com/RedHatInsights/notifications-backend/pull/478/files#diff-c2092ee52cefc16270b72e1ae4f060f48567699e6cdb8eb2317d1e66c0f2d22fR348-R352

Reply:
Isn't the new API part of an OpenAPI JSON file?

Reply:
It is part of the internal openapi. The types are build from the java types in this case, so we don't have to put `@Schema` in here